app-core/src/main/java/pm/gnosis/android/app/authenticator/util/ERC20.kt

```diff
@@ -1,5 +1,7 @@
 package pm.gnosis.android.app.authenticator.util
 
+import pm.gnosis.utils.isSolidityMethod
+import pm.gnosis.utils.removeSolidityMethodPrefix
 import java.math.BigInteger
 
 object ERC20 {
```

Comment:
Nothing related to this PR but this should be replaced in the future by the generated ERC20 ABI wrapper 😉